dev/release/source/Dockerfile

```diff
@@ -19,33 +19,14 @@ FROM ubuntu:18.04
 
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt-get update -y -q && \
```

Comment:
I've removed the libraries for building Arrow C++, because just runs `c_glib/autogen.sh`.